src/concepts/services/format.md

```diff
@@ -1,6 +1,41 @@
 # Format
 
-{% hint style="info" %}
-`Contribute to our docs:` [Explain the format module in depth](https://github.com/yewstack/docs/issues/24)
-{% endhint %}
+## Introduction
+Yew provides the format module to make it easy to convert from Rust types to common data formats (and vice versa). Rust provides support for a number of different data formats. Some of these formats are binary-only (e.g. CBOR) while others support both text and binary (e.g. JSON). 
```

Comment:
I would also like if there was a usage section. The format module is not a service itself, but is necessary for using the `FetchService` and `WebSocketService`